app/views/content/life-as-a-teacher/change-careers/support-with-changing-careers/_article.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <p>If you're thinking about changing careers to become a teacher, knowing where to start and understanding what your options are can seem overwhelming. There's lots of dedicated support available to help you navigate the change.</p>
+    <h2 class="heading--box-blue">Get advice and guidance from an expert</h2>
+    <h3>Teacher training advisers</h3>
+    <p>Our experienced teacher training advisers offer free one-to-one support to thousands of people each year to support them through a career change into teaching. All of our advisers have years of teaching experience, so can also give practical advice and insights on what life in the classroom is like.</p>
+    <p>If you're not sure if teaching is right for you, they may know of support available that you're not aware of, and can give practical advice on how other career changers have successfully trained to be a teacher.</p>
+    <p>To sign up you'll need to have, or be studying for, a bachelor's degree with honours.</p>
+    <p>You can chat to your dedicated adviser by phone, text or email.</p>
+    <p>Find out more about <a href="/teacher-training-advisers">getting a teacher training adviser</a>.</p>
+    <h3>Now Teach</h3>
+    <p>If you have a degree, you may be able to get <a href="https://nowteach.org.uk/">support from the charity Now Teach</a>. They specifically help people get into teaching who are changing career.</p>
+    <h3>Events</h3>
+    <p>Anyone interested in teaching can attend a <a href="/events/about-get-into-teaching-events">Get Into Teaching event</a>. These relaxed and informative events provide a one-stop shop of information and support. They'll give you the opportunity to get expert advice and talk to teachers and local training providers.</p>
+  </section>
+  <section class="col col-720">
+    <p><%= render 'content/shared/quotes/quote_career_change_phil' %></p>
+    <h2 class="heading--box-blue">Applying for teacher training</h2>
+    <p>When you apply for teacher training, you'll need to include things like proof of qualifications, and a written personal statement explaining why you want to become a teacher.<p>
+    <p>To help show your qualifications, you can <a href="https://www.gov.uk/replacement-exam-certificate">get a certified statement of your exam results</a> if you did GCSEs, CSEs, O levels or A levels in the UK. If you have a degree, you should also be able to request a copy of your degree certificate from the organisation where you studied in the UK.</p>
```

Comment:
Wonder if we should say 'university or organisation' - took me a sec to work out that organisation includes unis too